tests/ludwig/utils/test_normalization.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import pandas as pd
+
+from ludwig.features.numerical_feature import NumericalBaseFeature
+
+def numerical_feature():
+    return {'name': 'norm_test' , 'type': 'numerical'}
+
+column = pd.Series([
+    2,
+    4,
+    6,
+    8,
+    10
+])
+
+feature_1 = NumericalBaseFeature(numerical_feature())
+feature_2 = NumericalBaseFeature(numerical_feature())
+
+def test_norm():
+    assert feature_1.get_feature_meta(
+        column, {'normalization': 'zscore'}
+    )['mean'] == 6
+    assert feature_2.get_feature_meta(
+        column, {'normalization': 'minmax'}
+    )['min'] == 2
+    assert feature_2.get_feature_meta(
+        column, {'normalization': 'minmax'}
+    )['max'] == 10
+
```

Comment:
This is great. But could you please add a couple of checks validating the normalized values?

Reply:
e.g. after zscore normalization, the first row should be close to -1.4144. You can use np.allclose to verify

Reply:
Added the tests. Used [Unbiased Estimator](https://stackoverflow.com/questions/24984178/different-std-in-pandas-vs-numpy)(Pandas) for standard deviation calculation.

Reply:
@msaisumanth , Is there anything more I have to add to the `tests`?

Reply:
looks good!